R/middleware.R

```diff
@@ -341,9 +350,11 @@ HandlerManager <- R6Class("HandlerManager",
           headers$'Content-Type' <- response$content_type
 
           response <- filter(req, response)
-          return(list(status=response$status,
-            body=response$content,
-            headers=headers))
+          return(list(
+            status = response$status,
+            body = if (head_request) "" else response$content,
```

Comment:
I think digging will be required -- it's getting added magically, and I'm pretty sure it's not from inside the Shiny code base. :(